lib/cli/src/commands/auth/login/mod.rs

```diff
@@ -20,7 +20,7 @@ enum AuthorizationState {
     UnknownMethod,
 }
 
-/// Subcommand for log in a user into Wasmer (using a browser or provided a token)
+/// Login into a wasmer.io-like registry (using a browser or by providing a token)
```

Comment:
This can be a bit confusing for the user (what is wasmer.io-like? What is a registry?)

```suggestion
/// Login into Wasmer (using a browser or by providing a token created in https://wasmer.io/settings/access-tokens)
```

Reply:
`wasmer login` had that weird phrasing. I pushed another commit to adjust that to use Wasmer instead of wasmer.io like registry as well.

Reply:
Great, thx